### ISSUE TITLE
Horizon Container seems to use wrong python env

### ISSUE THREAD

**Issue by traskat** [osism/issues#419]
Hello,

i just rolled out the yoga release of horizon with osism. The container cannot start because of this
```
Tue Jan 31 08:51:11.063622 2023] [wsgi:error] [pid 46:tid 140215661483776] [client 10.172.4.100:36340] Target WSGI script not found or unable to stat: /var/lib/kolla/venv/lib/python3.10
```
There is no python3.10 in the venv only 3.8

```
ls  /var/lib/kolla/venv/lib/
python3.8
```

thanks in advance

**Comment by berendt** [comment]
What is the tag of the Horizon Container?

**Comment by traskat** [comment]
```
quay.io/osism/horizon:yoga
```

**Comment by berendt** [comment]
I am looking at this. You can use Zed's container as a workaround.

Add the following line in your ``environments/configuration/images.yml`` and redeploy the Horizon service.

```
horizon_tag: zed
```

**Comment by berendt** [comment]
Looks like kolla-ansible uses a wrong Python version for the venv. 3.10 instead of 3.8. But only 3.8 is there.

Do you use Ubuntu 20.04 or Ubuntu 22.04 as distribution?

```
()[root@5a1d9da9ed57 ~]# /usr/bin/python3 --version
Python 3.8.10
()[root@5a1d9da9ed57 ~]# /var/lib/kolla/venv/bin/python3 --version
Python 3.8.10
```

**Comment by berendt** [comment]
@traskat  New kolla-ansible image is available in appr. 30 minutes. Please retry if the issue is fixed and if not please reopen this issue.

**Comment by flyersa** [comment]
Have to reopen this. The same issue came up again with the R6 release and antelope...

**Comment by osfrickler** [comment]
Well that's not the same issue, then. Please open a new issue with detailed information.

**Comment by flyersa** [comment]
yeah of course. I reopen it then again on the next release if you get your shit together...

**Comment by berendt** [comment]
> Have to reopen this. The same issue came up again with the R6 release and antelope... but now its reverse. Now its asking for 3.8 but only 3.10 is available ;)

Works like expected with >= 2023.1 builds. As of SCS R5 we only support Ubuntu 22.04 as OS (see https://github.com/SovereignCloudStack/release-notes/blob/main/Release4.md for further details)
